### ISSUE TITLE
Could not find a package configuration file provided by "Torch"

### ISSUE THREAD

**Issue by marcoippolito** [pytorch/pytorch#29525]
I installed pytorch cpyonly in Ubuntu 18.04.02 through conda:

    (base) marco@pc:~$ conda list | grep torch
    cpuonly                   1.0                           0    pytorch
    facenet-pytorch           0.1.0                    pypi_0    pypi
    pytorch                   1.3.0               py3.7_cpu_0  [cpuonly]  pytorch
    torchaudio                0.4.0a0+38d1a9b          pypi_0    pypi
    torchfile                 0.1.0                    pypi_0    pypi
    torchtext                 0.4.0                    pypi_0    pypi
    torchvision               0.4.1                  py37_cpu  [cpuonly]  pytorch

    (base) marco@pc:~$ python3
    Python 3.7.3 (default, Mar 27 2019, 22:11:17) 
    [    GCC 7.3.0] :: Anaconda, Inc. on linux
    Type "help", "copyright", "credits" or "license" for more information.
    >>> import torch
    >>> import pytorch

When trying to build one of the examples of Cpp frontend:

    (base) marco@pc:~/pytorchCppFrontend/cppExamples/regression/build$ cmake  
    -DCMAKE_PREFIX_PATH=/path/to/libtorch ..
    CMake Error at CMakeLists.txt:6 (find_package):
      By not providing "FindTorch.cmake" in CMAKE_MODULE_PATH this project has
      asked CMake to find a package configuration file provided by "Torch", but
      CMake did not find one.

      Could not find a package configuration file provided by "Torch" with any of
      the following names:

        TorchConfig.cmake
        torch-config.cmake

      Add the installation prefix of "Torch" to CMAKE_PREFIX_PATH or set
      "Torch_DIR" to a directory containing one of the above files.  If "Torch"
      provides a separate development package or SDK, be sure it has been
    installed.

    -- Configuring incomplete, errors occurred!
    See also "/home/marco/pytorchCppFrontend/cppExamples/regression/build/CMakeFiles
    /CMakeOutput.log".

    (base) marco@pc:~/anaconda3$ find -name "torch-config.cmake"
    (base) marco@pc:~/anaconda3$ find -name "TorchConfig.cmake"
    ./pkgs/pytorch-1.2.0-cuda100py37h938c94c_0/lib/python3.7/site-packages/torch/share/cmake
    /Torch/TorchConfig.cmake
    ./pkgs/pytorch-1.3.0-py3.7_cpu_0/lib/python3.7/site-packages/torch/share/cmake/Torch
    /TorchConfig.cmake
    ./pkgs/pytorch-1.2.0-py3.7_cpu_0/lib/python3.7/site-packages/torch/share/cmake/Torch
    /TorchConfig.cmake
    ./pkgs/pytorch-1.0.1-py3.6_cuda10.0.130_cudnn7.4.2_2/lib/python3.6/site-packages/torch/share
    /cmake/Torch/TorchConfig.cmake
    ./lib/python3.7/site-packages/torch/share/cmake/Torch/TorchConfig.cmake
    ./envs/env_name/lib/python3.6/site-packages/torch/share/cmake/Torch/TorchConfig.cmake

I added in CMakeLists.txt the following line:

    set(TORCH_BUILD_DIR "/home/marco/anaconda3/lib/python3.7/site-packages/torch/share
    /cmake/Torch")

But got the same error.

How to solve the problem?
I attached the CMakeOutput.log

Marco
[CMakeOutput.log](https://github.com/pytorch/pytorch/files/3828741/CMakeOutput.log)

cc @ezyang

**Comment by SsnL** [comment]
replace `/path/to/libtorch` to actual path?

**Comment by marcoippolito** [comment]
Thank you Wang @SsnL !!!! 

    (base) marco@pc:~/pytorchCppFrontend/cppExamples/regression/build$ cmake    
    -DCMAKE_PREFIX_PATH=/home/marco/anaconda3/lib/python3.7/site-packages/torch ..
    -- Looking for pthread.h
    -- Looking for pthread.h - found
    -- Looking for pthread_create
    -- Looking for pthread_create - not found
    -- Looking for pthread_create in pthreads
    -- Looking for pthread_create in pthreads - not found
    -- Looking for pthread_create in pthread
    -- Looking for pthread_create in pthread - found
    -- Found Threads: TRUE  
    -- Found torch: /home/marco/anaconda3/lib/python3.7/site-packages/torch/lib/libtorch.so  
    -- Configuring done
    -- Generating done
    -- Build files have been written to: /home/marco/pytorchCppFrontend/cppExamples/regression/build

    (base) marco@pc:~/pytorchCppFrontend/cppExamples/regression/build$ make
    Scanning dependencies of target regression
    [ 50%] Building CXX object CMakeFiles/regression.dir/regression.cpp.o
    [100%] Linking CXX executable regression
    [100%] Built target regression

    (base) marco@pc:~/pytorchCppFrontend/cppExamples/regression/build$ ./regression 
    Loss: 0.0008861 after 642 batches
    ==> Learned function:	y = -8.1365 x^4 6.258 x^3 9.4452 x^2 -3.92712 x^1 + 2.56638
    ==> Actual function:	y = -8.14084 x^4 6.22522 x^3 9.42976 x^2 -3.92396 x^1 + 2.57184
